src/components/common/Extensions/UserHub/partials/BalanceTab/partials/StreamsInfoRow/hooks.ts

```diff
@@ -3,51 +3,74 @@ import { useCallback, useEffect, useMemo, useState } from 'react';
 import { useAppContext } from '~context/AppContext/AppContext.ts';
```

Comment:
If this hook is now reusable, it needs to be moved from a specific component's folder into shared hooks